### ISSUE TITLE
news - groundbreaking festival image alt text spelling

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#116]
https://rrchnm.org/news/

Please check spelling for the alt text as VOU saying something other than a couple of the words there. 

Significant words: I'm thinking 'livestreamed' should be 'live-streamed' and the dates like 9.12 and 9.18 should probably be september 12th; september 18th respectively

![Screen Shot 2020-10-19 at 2 48 26 PM](https://user-images.githubusercontent.com/618307/96498606-4752f200-121a-11eb-8cff-d206ba6a6e18.png)

**Comment by kimisgold** [comment]
Fixed within the content.
